cmd/addpool.go

```diff
@@ -44,7 +44,6 @@ type addPoolCmd struct {
 	client           armhelpers.AKSEngineClient
 	locale           *gotext.Locale
 	nameSuffix       string
-	nodePoolIndex    int
```

Comment:
`nodePoolIndex` is (AFAICT) unused at this point, probably the easiest implementation is to just append the new pool to the end of the existing `AgentPoolProfiles` array. (There shouldn't be a need to keep track of any array indices)

Reply:
Looks fine to me.  I don't remember using it anywhere. 
Thanks